services/src/libs/sentry.js

```diff
@@ -34,6 +35,9 @@ class Sentry {
     const errorData = {
       level: data.log.level.toLowerCase(),
       logger: data.log.name,
+      user: {
+        uuid: data.uuid,
```

Comment:
:bug: I think, you probably meant to write `id: data.uuid`. https://docs.sentry.io/learn/context#capturing-the-user

I believe that once changed, this will fix https://issues.jenkins-ci.org/browse/JENKINS-53286 right?